social-network/src/App.css

```diff
@@ -0,0 +1,31 @@
+.App {
+  text-align: left;
+  display: block;
+  margin-left: 10px;
+  font-family: medium-content-sans-serif-font,-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen,Ubuntu,Cantarell,"Open Sans","Helvetica Neue",sans-serif;
```

Comment:
While you didn't have to use the exact font-family used on Medium, I like that you went to their site and saw how they were implementing their CSS. It's a great way to learn new styling methods and tricks 👍